### ISSUE TITLE
Installing on node 10 fails due to fs-extra 10.0.0 requiring node 12

### ISSUE THREAD

**Issue by EdBailey** [semantic-release/semantic-release#2052]
## Current behavior

Installing semantic-release on node 10 produces the error `fs-extra@10.0.0: The engine "node" is incompatible with this module. Expected version ">=12". Got "10.24.1"`

It looks like this was introduced in https://github.com/semantic-release/semantic-release/pull/1955

## Expected behavior

Installing on node 10 should work, per the [readme](https://github.com/semantic-release/semantic-release#requirements) 

## Environment

- **semantic-release** version: 17.4.4

**Comment by gr2m** [comment]
Hm the CI passed on 10.18.0. I fear as much as we try to avoid it, these changes will sneak in. You really shouldn't run your deployment scripts with a no longer maintained node version.

Would you like to send a pull request to revert the change? We should update the `renovate` settings to ignore the `fs-extra` package for now, and also update the README that while we try to keep supporting Node 10.8+ with the current semantic-release versions, it's unavoidable that some sub dependencies introduce dependencies that no longer work with Node 10, and that we recommend using Node LTS

**Comment by EdBailey** [comment]
I actually think bumping the minimum node version would be entirely reasonable. As you say, node 10 is no longer maintained. I just happened to be trying to update semantic-release on a project where i can't easily upgrade the node version right now.

Looking further into this, it looks like the same issue exists in the NPM plugin and I would assume others as well. I'm not really sure where to start on making pull requests to downgrade it across multiple plugins...

For now I have solved it in my project by adding the following to my package.json
```
"resolutions": {
  "fs-extra": "^9.0.0"
}
```

**Comment by gr2m** [comment]
yeah we plan on releasing a breaking version of semantic-release which will bump the minimal node version.
